extensions/wikia/Email/i18n/Comment.i18n.php

```diff
@@ -0,0 +1,18 @@
+<?php
+$messages = [];
+
+$messages['en'] = [
+	'emailext-articlecomment-summary' => 'People are talking about $1 on {{SITENAME}}!',
+	'emailext-blogcomment-summary' => 'People are talking about the $1 post on {{SITENAME}}!',
+	'emailext-comment-salutation' => 'Hi $1,',
+	'emailext-comment-link-label' => 'See this comment',
+	'emailext-comment-view-all' => "[$1 View all comments]",
```

Comment:
Ok, LOC-28 got ... lost in translation.  Here's the new ticket, LOC-32: https://wikia-inc.atlassian.net/browse/LOC-32